components/process.rst

```diff
@@ -350,6 +350,18 @@ instead::
     );
     $process->run();
 
+Using the prepared command lines
+--------------------------------
+
+This component also provides a way to use the process command with prepared using the double brackets notations:
+
+    use Symfony\Component\Process\Process;
+
+  $process = Process::fromShellCommandline('echo "$name"');
+  $process->run(null, array('name' => 'elsa'));
+
+Which means that you can use placeholder in order to have a process that can be changed only with the values and without changing the php code. The component will not escape the characters, you are responsible of doing  so.
```

Comment:
2 spaces before the last word

Reply:
I would put the whole text parts before the Code example and not talk again about „the component“